db/01_create_schema.sh

```
@@ -0,0 +1,3 @@
+# !/bin/sh
```

Shellcheck (0.10.0)

[error] Remove spaces between # and ! in the shebang.

(SC1115)